docs/security-considerations.rst

```diff
@@ -289,3 +289,18 @@ of "failsafe" mode, which, for example, disables most of the features, hands ove
 control to a fixed and trusted third party or just converts the contract into
 a simple "give me back my money" contract.
 
+
+Method visibility from mother contracts and ABI
+===============================================
+
+
+When compiling a contract that inherits from other contracts, the generated JSON ABI for the child contract does not contain all the methods from contracts in the upper hierarchy. However, that does not prevent public methods from being executed. You should take care of which methods can be executed. Use `internal` function modifier to control visibility.
```

Comment:
Do you have an example for this?

```
contract A {
  function f() {}
}

contract B is A {
  function g() {}
}

contract C is B {
  function h() {}
}
```

results correctly in

`
======= abi.sol:A =======
Contract JSON ABI 
[{"constant":false,"inputs":[],"name":"f","outputs":[],"payable":false,"type":"function"}]

======= abi.sol:B =======
Contract JSON ABI 
[{"constant":false,"inputs":[],"name":"f","outputs":[],"payable":false,"type":"function"},{"constant":false,"inputs":[],"name":"g","outputs":[],"payable":false,"type":"function"}]

======= abi.sol:C =======
Contract JSON ABI 
[{"constant":false,"inputs":[],"name":"f","outputs":[],"payable":false,"type":"function"},{"constant":false,"inputs":[],"name":"h","outputs":[],"payable":false,"type":"function"},{"constant":false,"inputs":[],"name":"g","outputs":[],"payable":false,"type":"function"}]
``